modulemd/v2/include/modulemd-2.0/modulemd-module-stream-v1.h

```diff
@@ -517,6 +567,18 @@ modulemd_module_stream_v1_remove_rpm_api (ModulemdModuleStreamV1 *self,
                                           const gchar *rpm);
 
 
+/**
+ * modulemd_module_stream_v1_clear_rpm_api:
+ * @self: (in): This #ModulemdModuleStreamV1 object.
+ *
+ * Remove all RPMs from the list of stable public API.
+ *
+ * Since: 2.5
+ */
+void
+modulemd_module_stream_v1_clear_rpm_api (ModulemdModuleStreamV1 *self);
```

Comment:
I'm torn on this, but for naming consistency it feels like function name should be in the plural form, `*_clear_rpm_apis()`.

Reply:
I struggled with that, but in the other cases, the plural was more obvious on the last word. In this case, I think the singular fits better.
As long as the verb (create) is consistent, I think it's probably fine.